sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_thread_client_async.py

```diff
@@ -183,6 +183,8 @@ def list_read_receipts(
     ) -> AsyncItemPaged[ChatMessageReadReceipt]:
         """Gets read receipts for a thread.
 
+        :keyword int results_per_page: The maximum number of chat message read receipts to be returned per page.
+        :keyword int skip: Skips chat message read receipts up to a specified position in response.
         :keyword callable cls: A custom type or function that will be passed the direct response
```

Comment:
We should remove references to `cls` from the method docstring - `cls` is actually being overridden on line 209, so if a customer were to provide this value in addition - it would error.
Eventually it would be ideal if we do support this parameter for customers - but that could be a more extensive fix. In the meantime, simply removing this documentation would help.

Reply:
Alternatively - if you want to support it now - I think you could update line 209 to something like:
```python
cls=kwargs.pop('cls', None) or lambda objs: [ChatMessageReadReceipt._from_generated(x) for x in objs]
```
But you would need to write a test to confirm this, as I may have it wrong :)

Reply:
This has been fixed 👍